src/components/topbar/Topbar.jsx

```diff
@@ -24,27 +28,70 @@ const Topbar = (props) => {
   const handleDBSelect = (dbName) => history.push(`/mission-control/projects/${projectID}/database/${dbName}`)
   
   const svgIcon = dbIcons(projects, projectID, selectedDB)
+  const content = (
+    <div className="popContent">
+      <p style={{marginBottom:"50px", fontWeight:"bold", fontSize:"16px"}}>Love Space Cloud? Help us spread the love!</p>
+      <Row align="middle">
+        <Col md={{ span:12,  offset:0 }} >
+          <a  href="https://github.com/spaceuptech/space-cloud" target="_blank">
+            <img src={githubOctocat} />
+            <p style={{marginTop:"20px", color:"rgba(0, 0, 0, 0.65)", fontWeight:"600", fontSize:"14px"}}>Star</p>
+          </a>
+        </Col>
+        <Col md={{ span:12, offset:0 }}>
+          <a href="https://twitter.com/intent/tweet?text=I%20just%20tried%20%23spacecloud%20and%20it%20is%20amazing&original_referer="
```

Comment:
Use this as the twitter url - https://twitter.com/intent/tweet?text=Just%20deployed%20%23SpaceCloud%20-%20an%20opensource%20Firebase%20%2B%20Heroku!%20by%20%40SpaceUpTech.%0Ahttps%3A//github.com/spaceuptech/space-cloud%0A%23graphql%20%23webdev